edilkamin/api.py

```diff
@@ -201,3 +201,148 @@ def set_target_temperature(token: str, mac_address: str, temperature: int) -> st
     return mqtt_command(
         token, mac_address, {"name": "enviroment_1_temperature", "value": temperature}
     )
+
+
+def device_info_get_fan_1_speed(info: typing.Dict) -> int:
+    """Get fans speed value from cached info."""
+    return info["status"]["fans"]["fan_1_speed"]
+
+
+def get_fan_1_speed(token: str, mac_address: str) -> int:
+    """Get fans speed value."""
+    info = device_info(token, mac_address)
+    return device_info_get_fan_1_speed(info)
+
+
+def set_fan_1_speed(token: str, mac_address: str, fan_1_speed: int) -> str:
+    """
+    Set fan 1 speed.
+    Return response string e.g. "Command 0123456789abcdef executed successfully".
+    """
+    return mqtt_command(token, mac_address, {"name": "fan_1_speed", "value": fan_1_speed})
+
+
+def device_info_get_fan_2_speed(info: typing.Dict) -> int:
+    """Get fans speed value from cached info."""
+    return info["status"]["fans"]["fan_2_speed"]
+
+
+def get_fan_2_speed(token: str, mac_address: str) -> int:
+    """Get fans speed value."""
+    info = device_info(token, mac_address)
+    return device_info_get_fan_2_speed(info)
+
+
+def set_fan_2_speed(token: str, mac_address: str, fan_2_speed: int) -> str:
+    info = device_info(token, mac_address)
+    fan_number_confiuguration = info["nvm"]["installer_parameters"]["fans_number"]
+    
+    if fan_number_confiuguration >= 2:
+        """
+        Set fan 2 speed.
+        Return response string e.g. "Command 0123456789abcdef executed successfully".
+        """
```

Comment:
The docstring should go after the function definition

Reply:
Ok done.